### DISCUSSION TITLE
Golang Api Library - UpdateRequiredStatusChecks function - Need Help

### DISCUSSION THREAD

**Discussion by pwalters04** [community/community#24534] [Apps, API and Webhooks]
<pre><code class="lang-auto">	statusCheck := github.RequiredStatusChecks{
		Strict:   false,
		Contexts: nil,
	}
	
statusCk,resp,err:=c.gh.Repositories.UpdateRequiredStatusChecks(context.Background(),ORG_OWNER,repoName,branchName,**&amp;statusCheck**)
</code></pre>
<p>The text in bold is giving me this error: Cannot use ‘&amp;statusCheck’ (type *RequiredStatusChecks) as type *RequiredStatusChecksRequest</p>
<p>Please help. I am still learning, the UpdateRequiredStatusChecks parameters are this:<br>
UpdateRequiredStatusChecks(ctx context.Context, owner, repo, branch string, sreq *RequiredStatusChecksRequest) (*RequiredStatusChecks, *Response, error){}</p>

**Comment by pwalters04** [comment, marked as answer]
<p>solved it. its was a mix struct type</p>

**Comment by ghost** [comment]
Need help to remove hacker devices from my account but I can still see unknown devices with full access to my account I have only have iPhone X ISO 16 other than that all devices are working for hackers
